### DISCUSSION TITLE
IP limiter in Marzban

### DISCUSSION THREAD

**Discussion by mmdzov** [Gozargah/Marzban#491] [Show and tell]
سلام 

ابزار [luIP-marzban ](https://github.com/mmdzov/luIP-marzban) یک ابزار محدود کننده IP در مرزبان هست و به نوعی ترافیک ورودی IP بلاک شده رو از طریق iptables برای مدت مشخص و قابل انعطاف مسدود می کنه همچنین کاربران متصل دچار هیچگونه مشکلی اعم از قطعی پروکسی نمیشن.

توضیحات کامل در [luIP-marzban](https://github.com/mmdzov/luIP-marzban) قرار داده شده همچنین شما می تونید از API این ابزار در جهت توسعه پروژه هاتون استفاده کنید.

**Comment by xmohammad1** [comment]
ممنون من تست کردم به خوبی کار میکنه

**Comment by gapochat** [comment]
من روی چند تا سرور تست کردم کار نکرد...

**Comment by Mjzaret** [comment]
زمانی که از اینترنت همراه به وایفای سوئیچ میکنیم بدون خاموش کردن وی پی ان باعث میشه آی پی وایفای ما در بلک لیست سرور قرار بگیره و باید مدت زمانی که تعیین کردیم منتظر باشیم تا از بلک لیست دربیاد
یکی از مشکلاتی بود که موقع تست کردن بهش برخوردم.

**Comment by jomertix** [comment]
I'm categorically against adding an ip limiter to the marzban panel. Due to the peculiarities of network and the delay in sending packets, situations constantly arise that it's impossible to track the correct number of connected ip addresses. None of the tools can solve this problem. Let luIP remain a separate app
